src/test/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesFactoryAdapterTest.java

```diff
@@ -46,6 +50,18 @@
 
     private Map<String, String> systemProperties = new HashMap<>();
 
+    /**
```

Comment:
Doesn't seem related to the main change but ok.

Reply:
fixed